httpexpect/importAPI/get_job_test.go

```diff
@@ -1,68 +1,117 @@
+	mgo "gopkg.in/mgo.v2"
 )
 
-// Get a job
 // Get information about a single job
-// 200 - Return a single jobs information
-func TestGetJob_ReturnsSingleJob(t *testing.T) {
+// 200 - Return a single job information
+func TestSuccessfullyGetAnImportJob(t *testing.T) {
+
+	if err := mongo.Teardown("imports", "imports", "id", jobID); err != nil {
+		if err != mgo.ErrNotFound {
+			log.ErrorC("Was unable to run test", err, nil)
+			os.Exit(1)
+		}
+	}
+
+	if err := mongo.Setup("imports", "imports", "id", jobID, validCreatedImportJobData); err != nil {
+		log.ErrorC("Was unable to run test", err, nil)
+		os.Exit(1)
+	}
+
 	importAPI := httpexpect.New(t, cfg.ImportAPIURL)
 
-	Convey("Given an existing job", t, func() {
-
-		expected := importAPI.POST("/jobs").WithBytes([]byte(validJSON)).
-			Expect().Status(http.StatusCreated).JSON().Object()
-		jobID := expected.Value("id").String().Raw()
-		// expectedJobID := expected.Value("id").String().Raw()
-		// expectedRecipe := expected.Value("recipe").String().Raw()
-		// expectedState := expected.Value("state").String().Raw()
-		// expectedFileAliasName := expected.Value("files").Array().Element(0).Object().Value("alias_name").String().Raw()
-		// expectedFileURL := expected.Value("files").Array().Element(0).Object().Value("url").String().Raw()
-		// expectedIntanceID := expected.Value("links").Object().Value("instances").Array().Element(0).Object().Value("id").String().Raw()
-		// expectedIntanceLink := expected.Value("links").Object().Value("instances").Array().Element(0).Object().Value("href").String().Raw()
-
-		fmt.Println("expected is: %v", expected)
-
-		Convey("Get a specific job", func() {
-
-			_ = importAPI.GET("/jobs/{id}", jobID).Expect().Status(http.StatusOK).JSON().Object()
-
-			// actualJobID := actual.Value("id").String().Raw()
-			// actualRecipe := actual.Value("recipe").String().Raw()
-			// actualState := actual.Value("state").String().Raw()
-			// actualFileAliasName := actual.Value("files").Array().Element(0).Object().Value("alias_name").String().Raw()
-			// actualFileURL := actual.Value("files").Array().Element(0).Object().Value("url").String().Raw()
-			// actualIntanceID := actual.Value("links").Object().Value("instances").Array().Element(0).Object().Value("id").String().Raw()
-			// actualIntanceLink := actual.Value("links").Object().Value("instances").Array().Element(0).Object().Value("href").String().Raw()
-
-			// So(actualJobID, ShouldEqual, expectedJobID)
-			// So(actualRecipe, ShouldEqual, expectedRecipe)
-			// So(actualState, ShouldEqual, expectedState)
-			// So(actualFileAliasName, ShouldEqual, expectedFileAliasName)
-			// So(actualFileURL, ShouldEqual, expectedFileURL)
-
-			// So(actualIntanceID, ShouldEqual, expectedIntanceID)
-			// So(actualIntanceLink, ShouldEqual, expectedIntanceLink)
+	// These tests needs to refine when authentication was handled in the code.
+	Convey("Get information about a single job", t, func() {
+
```

Comment:
Remove whitespace between Wrapped subtests, they are indented, no need for an empty line between them.